src/main/drivers/pwm_output.c

```diff
@@ -231,27 +192,87 @@ void pwmFastPwmMotorConfig(const timerHardware_t *timerHardware, uint8_t motorIn
         timerMhzCounter = MULTISHOT_TIMER_MHZ;
         pwmWritePtr = pwmWriteMultiShot;
         break;
+    case (PWM_TYPE_BRUSHED):
+        timerMhzCounter = PWM_BRUSHED_TIMER_MHZ;
+        pwmWritePtr = pwmWriteBrushed;
+        motorConfig->use_unsyncedPwm = true;
+        idlePulse = 0;
+        break;
+    case (PWM_TYPE_CONVENTIONAL):
```

Comment:
`enum` is suffixed with `_CONVENTIONAL` but function is suffixed with `Standard`. I suggest `enum` is renamed to `PWM_TYPE_STANDARD`.